### DISCUSSION TITLE
Which 🇺🇸 state is nicknamed The 🌊 State?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#944] [Galaxy Trivia] [answered]
**Bonus:** What happened to its name 3 years ago?

**Comment by tomerf** [comment, marked as answer]
Rhode Island is nicknamed the ocean state. In November 2020, "State of Rhode Island and Providence Plantations" dropped the "and Providence Plantations" part.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/tomerf?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Comment by peterbaccaro** [comment]
The U.S. state nicknamed "The Ocean State" is **Rhode Island**. Rhode Island is known for its extensive coastline and beautiful beaches, which contribute to its nickname. Despite its name, Rhode Island is not an island but rather a state located in the New England region of the United States.

The state was officially named the State of Rhode Island and Providence Plantations since the colonial era, but came to be commonly known as "Rhode Island". **Three years ago (in November 2020), the state's voters approved an amendment to the state constitution formally dropping "and Providence Plantations" from its full name**.
